vector print does not work with search layer

When searching for an address, the point is highlighted.

But this point is not printed..

works for points and polygons, but not lines (eg Search for syre ... afterwards zoom to full extent as there is a bug with the zoom).
then print. generates a serverside error

```
java.util.concurrent.ExecutionException: java.lang.IllegalArgumentException: java.lang.IllegalArgumentException: java.lang.IllegalArgumentException: No rules found in style.  Rules are json objects that have the key * or have the form: [ecql]
	at jsr166y.ForkJoinTask.get(ForkJoinTask.java:913) ~[jsr166y-1.7.0.jar:1.7.0]
	at org.mapfish.print.output.AbstractJasperReportOutputFormat.getJasperPrint(AbstractJasperReportOutputFormat.java:159) ~[AbstractJasperReportOutputFormat.class:na]
	at org.mapfish.print.output.AbstractJasperReportOutputFormat.print(AbstractJasperReportOutputFormat.java:109) ~[AbstractJasperReportOutputFormat.class:na]
	at org.mapfish.print.MapPrinter.print(MapPrinter.java:138) ~[MapPrinter.class:na]
	at org.mapfish.print.servlet.job.PrintJob$1.run(PrintJob.java:134) ~[PrintJob$1.class:na]
	at org.mapfish.print.servlet.job.FilePrintJob.withOpenOutputStream(FilePrintJob.java:50) ~[FilePrintJob.class:na]
	at org.mapfish.print.servlet.job.PrintJob.call(PrintJob.java:131) [PrintJob.class:na]
	at org.mapfish.print.servlet.job.PrintJob.call(PrintJob.java:56) [PrintJob.class:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:262) [na:1.7.0_75]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145) [na:1.7.0_75]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615) [na:1.7.0_75]
	at java.lang.Thread.run(Thread.java:745) [na:1.7.0_75]
Caused by: java.lang.IllegalArgumentException: java.lang.IllegalArgumentException: java.lang.IllegalArgumentException: No rules found in style.  Rules are json objects that have the key * or have the form: [ecql]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.7.0_75]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57) ~[na:1.7.0_75]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.7.0_75]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526) ~[na:1.7.0_75]
	at jsr166y.ForkJoinTask.getThrowableException(ForkJoinTask.java:504) ~[jsr166y-1.7.0.jar:1.7.0]
	at jsr166y.ForkJoinTask.get(ForkJoinTask.java:912) ~[jsr166y-1.7.0.jar:1.7.0]
	... 11 common frames omitted
Caused by: java.lang.IllegalArgumentException: java.lang.IllegalArgumentException: No rules found in style.  Rules are json objects that have the key * or have the form: [ecql]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.7.0_75]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57) ~[na:1.7.0_75]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.7.0_75]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526) ~[na:1.7.0_75]
	at jsr166y.ForkJoinTask.getThrowableException(ForkJoinTask.java:504) ~[jsr166y-1.7.0.jar:1.7.0]
	at jsr166y.ForkJoinTask.reportResult(ForkJoinTask.java:567) ~[jsr166y-1.7.0.jar:1.7.0]
	at jsr166y.ForkJoinTask.join(ForkJoinTask.java:611) ~[jsr166y-1.7.0.jar:1.7.0]
	at org.mapfish.print.processor.ProcessorDependencyGraph$ProcessorGraphForkJoinTask.compute(ProcessorDependencyGraph.java:206) ~[ProcessorDependencyGraph$ProcessorGraphForkJoinTask.class:na]
	at org.mapfish.print.processor.ProcessorDependencyGraph$ProcessorGraphForkJoinTask.compute(ProcessorDependencyGraph.java:161) ~[ProcessorDependencyGraph$ProcessorGraphForkJoinTask.class:na]
	at jsr166y.RecursiveTask.exec(RecursiveTask.java:64) ~[jsr166y-1.7.0.jar:1.7.0]
	at jsr166y.ForkJoinTask.doExec(ForkJoinTask.java:305) ~[jsr166y-1.7.0.jar:1.7.0]
	at jsr166y.ForkJoinWorkerThread.execTask(ForkJoinWorkerThread.java:575) ~[jsr166y-1.7.0.jar:1.7.0]
	at jsr166y.ForkJoinPool.scan(ForkJoinPool.java:755) ~[jsr166y-1.7.0.jar:1.7.0]
	at jsr166y.ForkJoinPool.work(ForkJoinPool.java:617) ~[jsr166y-1.7.0.jar:1.7.0]
	at jsr166y.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:369) ~[jsr166y-1.7.0.jar:1.7.0]
Caused by: java.lang.IllegalArgumentException: No rules found in style.  Rules are json objects that have the key * or have the form: [ecql]
	at org.mapfish.print.map.style.json.MapfishJsonStyleVersion2.parseStyle(MapfishJsonStyleVersion2.java:111) ~[MapfishJsonStyleVersion2.class:na]
	at org.mapfish.print.map.style.json.MapfishJsonStyleParserPlugin$Versions$2.parseStyle(MapfishJsonStyleParserPlugin.java:440) ~[MapfishJsonStyleParserPlugin$Versions$2.class:na]
	at org.mapfish.print.map.style.json.MapfishJsonStyleParserPlugin.tryLoadJson(MapfishJsonStyleParserPlugin.java:488) ~[MapfishJsonStyleParserPlugin.class:na]
	at org.mapfish.print.map.style.json.MapfishJsonStyleParserPlugin.parseStyle(MapfishJsonStyleParserPlugin.java:462) ~[MapfishJsonStyleParserPlugin.class:na]
	at org.mapfish.print.map.style.StyleParser.loadStyle(StyleParser.java:65) ~[StyleParser.class:na]
	at org.mapfish.print.map.geotools.AbstractFeatureSourceLayerPlugin$1.load(AbstractFeatureSourceLayerPlugin.java:105) ~[AbstractFeatureSourceLayerPlugin$1.class:na]
	at org.mapfish.print.map.geotools.AbstractFeatureSourceLayerPlugin$1.load(AbstractFeatureSourceLayerPlugin.java:85) ~[AbstractFeatureSourceLayerPlugin$1.class:na]
	at org.mapfish.print.map.geotools.AbstractFeatureSourceLayer.getLayers(AbstractFeatureSourceLayer.java:94) ~[AbstractFeatureSourceLayer.class:na]
	at org.mapfish.print.map.geotools.AbstractGeotoolsLayer.render(AbstractGeotoolsLayer.java:97) ~[AbstractGeotoolsLayer.class:na]
	at org.mapfish.print.processor.map.CreateMapProcessor.createLayerGraphics(CreateMapProcessor.java:224) ~[CreateMapProcessor.class:na]
	at org.mapfish.print.processor.map.CreateMapProcessor.execute(CreateMapProcessor.java:157) ~[CreateMapProcessor.class:na]
	at org.mapfish.print.processor.map.CreateMapProcessor.execute(CreateMapProcessor.java:100) ~[CreateMapProcessor.class:na]
	at org.mapfish.print.processor.ProcessorGraphNode$ProcessorNodeForkJoinTask.compute(ProcessorGraphNode.java:210) ~[ProcessorGraphNode$ProcessorNodeForkJoinTask.class:na]
	at org.mapfish.print.processor.ProcessorGraphNode$ProcessorNodeForkJoinTask.compute(ProcessorGraphNode.java:187) ~[ProcessorGraphNode$ProcessorNodeForkJoinTask.class:na]
	at jsr166y.RecursiveTask.exec(RecursiveTask.java:64) ~[jsr166y-1.7.0.jar:1.7.0]
	at jsr166y.ForkJoinTask.doExec(ForkJoinTask.java:305) ~[jsr166y-1.7.0.jar:1.7.0]
	at jsr166y.ForkJoinWorkerThread.execTask(ForkJoinWorkerThread.java:575) ~[jsr166y-1.7.0.jar:1.7.0]
	at jsr166y.ForkJoinPool.scan(ForkJoinPool.java:733) ~[jsr166y-1.7.0.jar:1.7.0]
	... 2 common frames omitted
17:16:36.950 [PrintJobManager-1] DEBUG o.mapfish.print.servlet.job.PrintJob - Print Job 7cac6419-a8e1-4acd-a184-901c707801fd@e28f83b5-68ab-4f3c-9afc-6f3219fad5b1 completed in 5777ms
17:16:38.375 [http-bio-0.0.0.0-8083-exec-10] ERROR o.m.print.servlet.BaseMapServlet - Error while processing request: java.lang.IllegalArgumentException: No rules found in style.  Rules are json objects that have the key * or have the form: [ecql]

```

When printing this map:

http://devv3.geoportail.lu/main/theme/main?version=3&zoom=16&X=681055&Y=6378608&lang=lb&layers=373-154-146-151&opacities=1-1-1-1&bgLayer=topo_bw_jpeg


I get an error too, even without highlight:

```
+ CreateScalebarProcessor
08:52:34.580 [PrintJobManager-1] INFO  o.mapfish.print.servlet.job.PrintJob - Error executing print job 9f32e2de-86f5-4ffa-9506-257087c8cfe8@e28f83b5-68ab-4f3c-9afc-6f3219fad5b1
spec:
	{"app":"geoportailv3","outputFormat":"pdf","layout":"A4 portrait","attributes":{"scale":5000,"requestHeaders":{"requestHeaders":{"content-type":["application/json; charset=UTF-8"],"connection":["Keep-Alive"],"accept-language":["fr-LU,en-US;q=0.7,en;q=0.3"],"host":["devv3.geoportail.lu"],"x-forwarded-for":["148.110.80.240"],"accept":["application/json, text/plain, */*"],"x-forwarded-host":["devv3.geoportail.lu"],"user-agent":["Mozilla/5.0 (Macintosh; Intel Mac OS X 10.10; rv:38.0) Gecko/20100101 Firefox/38.0"],"pragma":["no-cache"],"cache-control":["no-cache"],"content-length":["7546"],"accept-encoding":["gzip, deflate"],"referer":["http://devv3.geoportail.lu/main/theme/main?version=3&zoom=16&X=681055&Y=6378608&lang=lb&layers=373-154-146-151&opacities=1-1-1-1&bgLayer=topo_bw_jpeg"],"x-forwarded-server":["devv3.geoportal.lu"]}},"legend":null,"name":"","map":{"center":[681055,6378608],"scale":7715.837114737902,"rotation":0,"layers":[{"style":{"version":2},"type":"geojson","geoJson":{"features":[],"type":"FeatureCollection"}},{"customParams":{"TRANSPARENT":true},"imageFormat":"png","layers":["mobiliteit_bikeshare"],"opacity":1,"baseURL":"http://devv3.geoportail.lu/main/wsgi/ogcproxywms","type":"wms"},{"customParams":{"TRANSPARENT":true},"imageFormat":"png","layers":["category"],"opacity":1,"baseURL":"http://ws.geoportail.lu/mymaps?category_id=991","type":"wms"},{"customParams":{"TRANSPARENT":true},"imageFormat":"png","layers":["arrets_bus"],"opacity":1,"baseURL":"http://devv3.geoportail.lu/main/wsgi/ogcproxywms","type":"wms"},{"style":"default","matrices":[{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[1,1],"tileSize":[256,256],"identifier":"00","scaleDenominator":5.590822640285715E8},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[2,2],"tileSize":[256,256],"identifier":"01","scaleDenominator":2.7954113201428574E8},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[4,4],"tileSize":[256,256],"identifier":"02","scaleDenominator":1.3977056600714287E8},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[8,8],"tileSize":[256,256],"identifier":"03","scaleDenominator":6.988528300357144E7},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[16,16],"tileSize":[256,256],"identifier":"04","scaleDenominator":3.494264150178572E7},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[32,32],"tileSize":[256,256],"identifier":"05","scaleDenominator":1.747132075089286E7},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[64,64],"tileSize":[256,256],"identifier":"06","scaleDenominator":8735660.375464285},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[128,128],"tileSize":[256,256],"identifier":"07","scaleDenominator":4367830.187714286},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[256,256],"tileSize":[256,256],"identifier":"08","scaleDenominator":2183915.093860714},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[512,512],"tileSize":[256,256],"identifier":"09","scaleDenominator":1091957.546932143},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[1024,1024],"tileSize":[256,256],"identifier":"10","scaleDenominator":545978.7734642858},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[2048,2048],"tileSize":[256,256],"identifier":"11","scaleDenominator":272989.38673285715},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[4096,4096],"tileSize":[256,256],"identifier":"12","scaleDenominator":136494.69336642858},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[8192,8192],"tileSize":[256,256],"identifier":"13","scaleDenominator":68247.34668321429},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[16384,16384],"tileSize":[256,256],"identifier":"14","scaleDenominator":34123.673341607144},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[32768,32768],"tileSize":[256,256],"identifier":"15","scaleDenominator":17061.836670785713},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[65536,65536],"tileSize":[256,256],"identifier":"16","scaleDenominator":8530.918335392857},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[131072,131072],"tileSize":[256,256],"identifier":"17","scaleDenominator":4265.459167714285},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[262144,262144],"tileSize":[256,256],"identifier":"18","scaleDenominator":2132.7295838500004},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[524288,524288],"tileSize":[256,256],"identifier":"19","scaleDenominator":1066.3647919250002}],"imageFormat":"image/png","matrixSet":"GLOBAL_WEBMERCATOR","layer":"addresses","dimensionParams":{},"baseURL":"http://wmts.geoportail.lu/mapproxy_4_v3/wmts/addresses/{TileMatrixSet}/{TileMatrix}/{TileCol}/{TileRow}.png","dimensions":[],"type":"WMTS","version":"1.0.0","requestEncoding":"REST"},{"style":"default","matrices":[{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[1,1],"tileSize":[256,256],"identifier":"00","scaleDenominator":5.590822640285715E8},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[2,2],"tileSize":[256,256],"identifier":"01","scaleDenominator":2.7954113201428574E8},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[4,4],"tileSize":[256,256],"identifier":"02","scaleDenominator":1.3977056600714287E8},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[8,8],"tileSize":[256,256],"identifier":"03","scaleDenominator":6.988528300357144E7},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[16,16],"tileSize":[256,256],"identifier":"04","scaleDenominator":3.494264150178572E7},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[32,32],"tileSize":[256,256],"identifier":"05","scaleDenominator":1.747132075089286E7},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[64,64],"tileSize":[256,256],"identifier":"06","scaleDenominator":8735660.375464285},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[128,128],"tileSize":[256,256],"identifier":"07","scaleDenominator":4367830.187714286},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[256,256],"tileSize":[256,256],"identifier":"08","scaleDenominator":2183915.093860714},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[512,512],"tileSize":[256,256],"identifier":"09","scaleDenominator":1091957.546932143},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[1024,1024],"tileSize":[256,256],"identifier":"10","scaleDenominator":545978.7734642858},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[2048,2048],"tileSize":[256,256],"identifier":"11","scaleDenominator":272989.38673285715},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[4096,4096],"tileSize":[256,256],"identifier":"12","scaleDenominator":136494.69336642858},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[8192,8192],"tileSize":[256,256],"identifier":"13","scaleDenominator":68247.34668321429},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[16384,16384],"tileSize":[256,256],"identifier":"14","scaleDenominator":34123.673341607144},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[32768,32768],"tileSize":[256,256],"identifier":"15","scaleDenominator":17061.836670785713},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[65536,65536],"tileSize":[256,256],"identifier":"16","scaleDenominator":8530.918335392857},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[131072,131072],"tileSize":[256,256],"identifier":"17","scaleDenominator":4265.459167714285},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[262144,262144],"tileSize":[256,256],"identifier":"18","scaleDenominator":2132.7295838500004},{"topLeftCorner":[-2.00375083428E7,2.00375083428E7],"matrixSize":[524288,524288],"tileSize":[256,256],"identifier":"19","scaleDenominator":1066.3647919250002}],"imageFormat":"image/jpeg","matrixSet":"GLOBAL_WEBMERCATOR","layer":"topo_bw_jpeg","dimensionParams":{},"baseURL":"http://wmts.geoportail.lu/mapproxy_4_v3/wmts/topo_bw_jpeg/{TileMatrixSet}/{TileMatrix}/{TileCol}/{TileRow}.jpeg","dimensions":[],"type":"WMTS","version":"1.0.0","requestEncoding":"REST"}],"projection":"EPSG:3857","dpi":96},"lang":"fr","url":"http://g-o.lu/v3/2Uch","qrimage":"http://devv3.geoportail.lu/main/wsgi/qr?url=http://g-o.lu/v3/2Uch"}}
java.util.concurrent.ExecutionException: java.lang.IllegalArgumentException: java.lang.IllegalArgumentException: java.lang.IllegalArgumentException: No rules found in style.  Rules are json objects that have the key * or have the form: [ecql]
	at jsr166y.ForkJoinTask.get(ForkJoinTask.java:913) ~[jsr166y-1.7.0.jar:1.7.0]
	at org.mapfish.print.output.AbstractJasperReportOutputFormat.getJasperPrint(AbstractJasperReportOutputFormat.java:159) ~[AbstractJasperReportOutputFormat.class:na]
	at org.mapfish.print.output.AbstractJasperReportOutputFormat.print(AbstractJasperReportOutputFormat.java:109) ~[AbstractJasperReportOutputFormat.class:na]
	at org.mapfish.print.MapPrinter.print(MapPrinter.java:138) ~[MapPrinter.class:na]
	at org.mapfish.print.servlet.job.PrintJob$1.run(PrintJob.java:134) ~[PrintJob$1.class:na]
	at org.mapfish.print.servlet.job.FilePrintJob.withOpenOutputStream(FilePrintJob.java:50) ~[FilePrintJob.class:na]
	at org.mapfish.print.servlet.job.PrintJob.call(PrintJob.java:131) [PrintJob.class:na]
	at org.mapfish.print.servlet.job.PrintJob.call(PrintJob.java:56) [PrintJob.class:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:262) [na:1.7.0_75]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145) [na:1.7.0_75]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615) [na:1.7.0_75]
	at java.lang.Thread.run(Thread.java:745) [na:1.7.0_75]
Caused by: java.lang.IllegalArgumentException: java.lang.IllegalArgumentException: java.lang.IllegalArgumentException: No rules found in style.  Rules are json objects that have the key * or have the form: [ecql]
	at sun.reflect.GeneratedConstructorAccessor44.newInstance(Unknown Source) ~[na:na]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.7.0_75]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526) ~[na:1.7.0_75]
	at jsr166y.ForkJoinTask.getThrowableException(ForkJoinTask.java:504) ~[jsr166y-1.7.0.jar:1.7.0]
	at jsr166y.ForkJoinTask.get(ForkJoinTask.java:912) ~[jsr166y-1.7.0.jar:1.7.0]
	... 11 common frames omitted
Caused by: java.lang.IllegalArgumentException: java.lang.IllegalArgumentException: No rules found in style.  Rules are json objects that have the key * or have the form: [ecql]
	at sun.reflect.GeneratedConstructorAccessor44.newInstance(Unknown Source) ~[na:na]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.7.0_75]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526) ~[na:1.7.0_75]
	at jsr166y.ForkJoinTask.getThrowableException(ForkJoinTask.java:504) ~[jsr166y-1.7.0.jar:1.7.0]
	at jsr166y.ForkJoinTask.reportResult(ForkJoinTask.java:567) ~[jsr166y-1.7.0.jar:1.7.0]
	at jsr166y.ForkJoinTask.join(ForkJoinTask.java:611) ~[jsr166y-1.7.0.jar:1.7.0]
	at org.mapfish.print.processor.ProcessorDependencyGraph$ProcessorGraphForkJoinTask.compute(ProcessorDependencyGraph.java:206) ~[ProcessorDependencyGraph$ProcessorGraphForkJoinTask.class:na]
	at org.mapfish.print.processor.ProcessorDependencyGraph$ProcessorGraphForkJoinTask.compute(ProcessorDependencyGraph.java:161) ~[ProcessorDependencyGraph$ProcessorGraphForkJoinTask.class:na]
	at jsr166y.RecursiveTask.exec(RecursiveTask.java:64) ~[jsr166y-1.7.0.jar:1.7.0]
	at jsr166y.ForkJoinTask.doExec(ForkJoinTask.java:305) ~[jsr166y-1.7.0.jar:1.7.0]
	at jsr166y.ForkJoinWorkerThread.execTask(ForkJoinWorkerThread.java:575) ~[jsr166y-1.7.0.jar:1.7.0]
	at jsr166y.ForkJoinPool.scan(ForkJoinPool.java:755) ~[jsr166y-1.7.0.jar:1.7.0]
	at jsr166y.ForkJoinPool.work(ForkJoinPool.java:617) ~[jsr166y-1.7.0.jar:1.7.0]
	at jsr166y.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:369) ~[jsr166y-1.7.0.jar:1.7.0]
Caused by: java.lang.IllegalArgumentException: No rules found in style.  Rules are json objects that have the key * or have the form: [ecql]
	at org.mapfish.print.map.style.json.MapfishJsonStyleVersion2.parseStyle(MapfishJsonStyleVersion2.java:111) ~[MapfishJsonStyleVersion2.class:na]
	at org.mapfish.print.map.style.json.MapfishJsonStyleParserPlugin$Versions$2.parseStyle(MapfishJsonStyleParserPlugin.java:440) ~[MapfishJsonStyleParserPlugin$Versions$2.class:na]
	at org.mapfish.print.map.style.json.MapfishJsonStyleParserPlugin.tryLoadJson(MapfishJsonStyleParserPlugin.java:488) ~[MapfishJsonStyleParserPlugin.class:na]
	at org.mapfish.print.map.style.json.MapfishJsonStyleParserPlugin.parseStyle(MapfishJsonStyleParserPlugin.java:462) ~[MapfishJsonStyleParserPlugin.class:na]
	at org.mapfish.print.map.style.StyleParser.loadStyle(StyleParser.java:65) ~[StyleParser.class:na]
	at org.mapfish.print.map.geotools.AbstractFeatureSourceLayerPlugin$1.load(AbstractFeatureSourceLayerPlugin.java:105) ~[AbstractFeatureSourceLayerPlugin$1.class:na]
	at org.mapfish.print.map.geotools.AbstractFeatureSourceLayerPlugin$1.load(AbstractFeatureSourceLayerPlugin.java:85) ~[AbstractFeatureSourceLayerPlugin$1.class:na]
	at org.mapfish.print.map.geotools.AbstractFeatureSourceLayer.getLayers(AbstractFeatureSourceLayer.java:94) ~[AbstractFeatureSourceLayer.class:na]
	at org.mapfish.print.map.geotools.AbstractGeotoolsLayer.render(AbstractGeotoolsLayer.java:97) ~[AbstractGeotoolsLayer.class:na]
	at org.mapfish.print.processor.map.CreateMapProcessor.createLayerGraphics(CreateMapProcessor.java:224) ~[CreateMapProcessor.class:na]
	at org.mapfish.print.processor.map.CreateMapProcessor.execute(CreateMapProcessor.java:157) ~[CreateMapProcessor.class:na]
	at org.mapfish.print.processor.map.CreateMapProcessor.execute(CreateMapProcessor.java:100) ~[CreateMapProcessor.class:na]
	at org.mapfish.print.processor.ProcessorGraphNode$ProcessorNodeForkJoinTask.compute(ProcessorGraphNode.java:210) ~[ProcessorGraphNode$ProcessorNodeForkJoinTask.class:na]
	at org.mapfish.print.processor.ProcessorGraphNode$ProcessorNodeForkJoinTask.compute(ProcessorGraphNode.java:187) ~[ProcessorGraphNode$ProcessorNodeForkJoinTask.class:na]
	at jsr166y.RecursiveTask.exec(RecursiveTask.java:64) ~[jsr166y-1.7.0.jar:1.7.0]
	at jsr166y.ForkJoinTask.doExec(ForkJoinTask.java:305) ~[jsr166y-1.7.0.jar:1.7.0]
	at jsr166y.ForkJoinWorkerThread.execTask(ForkJoinWorkerThread.java:575) ~[jsr166y-1.7.0.jar:1.7.0]
	at jsr166y.ForkJoinPool.scan(ForkJoinPool.java:733) ~[jsr166y-1.7.0.jar:1.7.0]
	... 2 common frames omitted
08:52:34.581 [PrintJobManager-1] DEBUG o.mapfish.print.servlet.job.PrintJob - Print Job 9f32e2de-86f5-4ffa-9506-257087c8cfe8@e28f83b5-68ab-4f3c-9afc-6f3219fad5b1 completed in 2072ms
08:52:35.622 [http-bio-0.0.0.0-8083-exec-4] ERROR o.m.print.servlet.BaseMapServlet - Error while processing request: java.lang.IllegalArgumentException: No rules found in style.  Rules are json objects that have the key * or have the form: [ecql]
```

--- a/geoportailv3/static/js/print/printdirective.js
+++ b/geoportailv3/static/js/print/printdirective.js
@@ -442,7 +442,9 @@ app.PrintController.prototype.print = function() {
         var resolution = map.getView().getResolution();
         goog.asserts.assert(goog.isDef(resolution));
         this.print_.encodeLayer(layers, this.vectorOverlayLayer_, resolution);
-        spec.attributes.map.layers.unshift(layers[0]);
+        if (layers.length > 0) {
+          spec.attributes.map.layers.unshift(layers[0]);
+        }
 
         // create print report
         this.print_.createReport(spec, /** @type {angular.$http.Config} */ ({

